zeton/templates/child_firstname_change.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% block content %}
+
+    <div class="col col-sm-6">
+        <div class="text-center">
+            <h2>{{ user_data.firstname }},</h2>
+            <h2>aktualne imię dziecka to: {{ child.firstname }}</h2>
+
+            {% if messages %}
+                <div class="alert alert-dark">{{ messages[0] }}</div>
+            {% endif %}
+
+            <form action="/api/settings/{{ child.id }}/set_firstname" method="post">
+                <div class="form-group">
+                    <label for="inputFirstname">Wpisz poniżej nowe imię:</label>
+                    <br>
+                    <input type="text" class="form-control" name="new_firstname" id="inputFirstname"
+                           placeholder="Nowe imię">
+                </div>
+                <br>
+                <div class="form-group button">
+                    <button type="submit" class="btn btn-info">Prześlij</button>
+                </div>
+                <div>
+                    <div class="text-center">
+
+                        <form>
+                            <div class="form-group button">
+
+                                <a class="btn btn-info" href="/">Strona główna</a>
+
+                                <a class="btn btn-info" href="/logout">Wyloguj</a>
+
+                                <input class="btn btn-info" value="Wróć" size="5" onclick="history.go(-1);"/>
```

Comment:
@visse0001 Ten przycisk nie działa zbyt przyjaźnie- używając go do powrotu na poprzednią stronę, cofamy się w historii i po zmianie imienia nadal widnieje to stare, a żeby zmiany były widoczne trzeba odświeżyć stronę.

Reply:
Racja.
To przycisk powielony z innego issue.
Poprawiłam to.